lib/FreeRTOS/mqtt-agent/mqtt_agent.h

```diff
@@ -341,7 +364,7 @@ MQTTStatus_t MQTTAgent_Publish( MQTTAgentContext_t * pMqttAgentContext,
  * command to be posted to the MQTT agent, should the agent's event queue be
  * full.  Tasks wait in the Blocked state so don't use any CPU time.
  *
- * @return `MQTTSuccess` if the command was posted to the MQTT agents event queue.
+ * @return `MQTTSuccess` if the command was posted to the MQTT agent's event queue.
  * Otherwise an enumerated error code.
  */
 MQTTStatus_t MQTTAgent_TriggerProcessLoop( MQTTAgentContext_t * pMqttAgentContext,
```

Comment:
missing declare tag?

Reply:
I've been ignoring this function since I've been wanting to rename it to `MQTTAgent_ProcessLoop` and change the parameters to match the other function, but I thought I'd do that in another PR. I added the declare tag for now